lib/results_validators/persons_validator.rb

```diff
@@ -74,26 +95,18 @@ def run_validation(validator_data)
         end
         duplicate_newcomer_names = []
         without_wca_id.each do |p|
-          if p.dob.month == 1 && p.dob.day == 1
-            @warnings << ValidationWarning.new(:persons, competition.id,
-                                               DOB_0101_WARNING,
-                                               name: p.name)
-          end
           if p.gender.blank?
             @warnings << ValidationWarning.new(:persons, competition.id,
                                                EMPTY_GENDER_WARNING,
                                                name: p.name)
           end
-          # Competitor less than 3 years old are extremely rare, so we'd better check these birthdate are correct.
```

Comment:
The information contained in this comment is lost.

Reply:
(I think you should copy it to the new code snippet)

Reply:
added to the new code, thank you.